pytta/rooms.py

```diff
@@ -28,13 +33,267 @@
 
 import numpy as np
 import matplotlib.pyplot as plt
-from numba import njit
+# from numba import njit
 from pytta import SignalObj, OctFilter, Analysis, ImpulsiveResponse
-from pytta.utils import fractional_octave_frequencies as FOF
+from pytta.utils import fractional_octave_frequencies as FOF, freq_to_band
 import traceback
 import copy as cp
 
 
+class RoomParameters(Analysis):
```

Comment:
To keep the toolbox's organization this class should go to the classes sub-package. rooms still a good name for the module.

Reply:
So you propose to move all the rooms.py module inside the classes sub-package?

Reply:
Not all the current rooms.py, but only the class. The functions could stay here.